pyam/_ops.py

```diff
@@ -104,7 +104,7 @@ def _op_data(df, name, method, axis, fillna=None, args=(), ignore_units=False, *
             )
 
     # merge all args and kwds that are based on `df._data` to apply fillna
-    if fillna:
+    if fillna is not None:  # can not check only if fillna: as 0s arent caught
```

Comment:
I think that the extra comment makes the code more complicated without helping to understand the logic - and the test would catch it if anyone ever tried to simplify that line...
```suggestion
    if fillna is not None:
```